packages/tempus/src/Tempus_IntegratorBasicOld_impl.hpp

```diff
@@ -387,8 +387,8 @@ bool IntegratorBasicOld<Scalar>::advanceTime()
     startIntegrator();
     integratorObserver_->observeStartIntegrator(*this);
 
-    while (integratorStatus_ == WORKING and
-        timeStepControl_->timeInRange (solutionHistory_->getCurrentTime()) and
+    while (integratorStatus_ == WORKING &&
```

Comment:
what is the reasoning behind making these changes?

Reply:
See #9018.  Some how these changed back.